node/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "opent2t-lib",
+  "version": "0.1.0",
+  "description": "Open Translators to Things",
+  "main": "build/lib",
+  "typings": "build/lib",
+  "scripts": {
+    "build": "tsc",
+    "lint": "tslint --force lib/**/*.ts",
+    "test": "tsc && ava --verbose"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/opent2t/opent2t.git"
+  },
+  "author": "opent2t@microsoft.com",
```

Comment:
>opent2t@microsoft.com [](start = 13, length = 21)

Is it OK to give MS alias ? I don't remember whether we are doing this currently.